docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb

```diff
@@ -232,81 +226,123 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "id": "convinced-theorem",
+   "execution_count": null,
+   "id": "3b96aa3e",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Server starts at host 127.0.0.1, port 5000\n"
+      " * Serving Flask app 'tinyms.serving.server.server' (lazy loading)\n",
+      "\n",
+      " * Environment: production\n",
+      "\n",
+      "   WARNING: This is a development server. Do not use it in a production deployment.\n",
+      "\n",
+      "   Use a production WSGI server instead.\n",
+      "\n",
+      " * Debug mode: off\n",
+      "\n"
      ]
     }
    ],
    "source": [
-    "start_server()"
+    "server = Server(serving_path=\"/home/jovyan/tinyms/serving/\")\n",
+    "server.start_server()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec82a869",
+   "metadata": {},
+   "source": [
+    "出现如上提示说明服务器已启动且正在运行。"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7fa5fc91",
+   "metadata": {},
+   "source": [
+    "服务器启动后，我们要到菜单栏点击`File`=>`new Notebook`创建一个新的jupyter的文件，然后继续完成客户端的操作。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ef79fe68",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from PIL import Image\n",
+    "\n",
+    "from tinyms.vision import ImageViewer\n",
+    "from tinyms.serving import Server, Client"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "european-wallpaper",
+   "id": "7bded060",
    "metadata": {},
    "source": [
     "### 6. 推理\n",
     "\n",
     "#### 6.1 上传图片\n",
     "\n",
-    "用户需要上传一张0~9之间的数字图片作为输入。如果使用命令行终端，可以使用'scp'或者'wget'获取图片，如果使用Jupyter，点击菜单右上方的'Upload'按钮并且选择上传的图片。本教程中使用的图片可以点击[这里](https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png)进行下载，将图片保存在根目录下，重命名为'7.png'（或者任何用户喜欢的名字）\n",
+    "用户需要上传一张0~9之间的数字图片作为输入。如果使用命令行终端，可以使用'scp'或者'wget'获取图片，如果使用Jupyter，点击菜单右上方的'Upload'按钮并且选择上传的图片。本教程中使用的图片可以点击[这里](https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png)进行下载，将图片保存在根目录下，重命名为'7.png'（或者任何用户喜欢的名字）。\n",
```

Comment:
Hi,@xinwenh.In order to unify the specification, it is recommended to change 'scp' and 'wget' to `scp` and `wget`